app/jobs/generic_job.rb

```diff
@@ -14,6 +14,9 @@
 class GenericJob < ApplicationJob
   # A somewhat easy to understand and informative time stamp format
   TIME_FORMAT = '%Y-%m-%d %H:%M%P'
+  DO_NOT_ALERT = {
+    'DescriptiveMetadataImportJob' => ['Cocina::Models::ValidationError']
```

Comment:
How about rescuing this in the `DescriptiveMetadataImportJob` rather than add code to the base class just for the one subclass?

Reply:
@jcoyne,  I see that we already rescue in `DescriptiveImport` (https://github.com/sul-dlss/argo/blob/17cef1d40cde1f2c0521be6f0ceca65e863c9038/app/services/description_import.rb#L28) and that bubbles up into `GenericJob` where the alert is sent... 

I'm not clear on how to avoid the HB alert, which I think but could be wrong, is rescued in https://github.com/sul-dlss/argo/blob/main/app/jobs/generic_job.rb#L136 with the `failure.call` at https://github.com/sul-dlss/argo/blob/main/app/jobs/descriptive_metadata_import_job.rb#L23

Should I refactor the error path and remove the `failure.call` in `DescriptiveImportJob`? Or will that impact the failure count reporting?

Thanks.

Reply:
There is no way for that rescued error to "bubble up".  It must have been from a different location.  Don't remove the failure.call, or you won't get proper counts.

Reply:
It's the `cocina_object.new(description: description)` that is raising the error, not the DescriptionImport

Reply:
Done. Thanks.